avocado/utils/kernel.py

```diff
@@ -108,9 +108,15 @@ def uncompress(self):
         else:
             raise Exception("Unable to find the tarball")
 
-    def configure(self):
+    def configure(self, targets=('defconfig'), extra_configs=None):
```

Comment:
The tuple is not a mutable, so OK.

Reply:
Learned this in one of first contributions to Avocado ;)